dev-java/jmock/jmock-2.6.1-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
```

Comment:
please also add `MAVEN_ID`

Reply:
Please re-check.  Missing any information how to determine the exact MAVEN_ID
[Java_Developer_Guide](https://wiki.gentoo.org/wiki/Java_Developer_Guide) is lacking such information :-(

Reply:
sorry for that. `MAVEN_ID` is used by our tool called `java-ebuilder` to find matching dependencies. `MAVEN_ID` is just what it says, id of the package in maven repository. the template is`group_id:artifact_id:version`. i was just searching for the correct value for this version but i must admit i'm confused with jmock and their maven artifacts. anyway, [here](https://github.com/jmock-developers/jmock-library/tree/master) you can see what it looks like for their currentversion (in the maven section, <dependency>...). i suppose in our case it should be `MAVEN_ID="org.jmock:jmock-junit4:2.6.1"` but it might aswell be `jmock-legacy` so really not sure. so leave it as it is, thanks and sorry for the confusion :-)

Reply:
Did you mean to keep what I added here?
And, thanks for the explaining the template. I'll try to use it whenever possible.

Reply:
please drop the following `MAVEN_ID`, that's not valid (the syntax is `group_id:artifact_id:version` and we use the version of the upstream, so in this case it would be `2.6.1`, without that revision number `-r1`). and as i wrote, i'm not even sure what is the correct `group_id` and `artifact_id` for this version.

Reply:
Dropped.  Sorry for misreading.